x/harvest/keeper/keeper.go

```diff
@@ -178,7 +182,70 @@ func (k Keeper) IterateClaimsByTypeAndDenom(ctx sdk.Context, depositType types.D
 	}
 }
 
+// GetDepositsByUser gets all deposits for an individual user
+func (k Keeper) GetDepositsByUser(ctx sdk.Context, user sdk.AccAddress) []types.Deposit {
```

Comment:
If we keep the architecture this way - `GetDepositsByUser` can be improved by iterating the denoms (params.MoneyMarkets[x].Denom) and calling `GetDeposit(ctx,user,denom, types.lp)` for each denom.

Reply:
Created issue https://github.com/Kava-Labs/kava/issues/705 for this